src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/MappingQualityZero.java

```diff
@@ -38,7 +39,11 @@
         }
         //NOTE: unlike other annotations, this one returns 0 if stratifiedPerReadAlleleLikelihoodMap is empty
         final long mq0 = stratifiedPerReadAlleleLikelihoodMap.values().stream().flatMap(llm -> llm.getLikelihoodReadMap().keySet().stream()).filter(r -> r.getMappingQuality() == 0).count();
-        return Collections.singletonMap(getKeyNames().get(0), String.format("%d", mq0));
+        return Collections.singletonMap(getKeyNames().get(0), formattedValue(mq0));
+    }
+
+    public static String formattedValue(long mq0) {
+        return String.format("%d", mq0);
```

Comment:
I don't think this trivial `String.format()` call deserves its own method, especially given that it's only called in 1 place.